native_client/ctcdecode/scorer.cpp

```diff
@@ -30,18 +30,16 @@ static const int32_t MAGIC = 'TRIE';
 static const int32_t FILE_VERSION = 4;
 
 Scorer::Scorer(double alpha,
-               double beta,
-               const std::string& lm_path,
-               const std::string& trie_path,
-               const Alphabet& alphabet)
-  : dictionary()
-  , language_model_()
-  , is_character_based_(true)
+               double beta)
+  : is_character_based_(true)
   , max_order_(0)
-  , alphabet_(alphabet)
 {
   reset_params(alpha, beta);
+}
 
+void
+Scorer::init(const std::string& lm_path, const std::string& trie_path)
```

Comment:
We pass `lm_path` and `trie_path` on `init` as well as on the constructor (below) ?